DrinkManagerConsole/DrinkManagerConsole.csproj

```diff
@@ -16,6 +16,11 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.7" />
+    <PackageReference Include="MockQueryable.Core" Version="3.1.3" />
```

Comment:
Those packages are not needed for DrinkManagerConsole project

Reply:
@marek-wilk probably you used "manage nuget packages" option for entire solution, not specific project, thats why packages were installed even for console app :)

Reply:
Removed them.